rav1e_js/src/encoder.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2020, The rav1e contributors. All rights reserved
+//
+// This source code is subject to the terms of the BSD 2 Clause License and
+// the Alliance for Open Media Patent License 1.0. If the BSD 2 Clause License
+// was not distributed with this source code in the LICENSE file, you can
+// obtain it at www.aomedia.org/license/software. If the Alliance for Open
+// Media Patent License 1.0 was not distributed with this source code in the
+// PATENTS file, you can obtain it at www.aomedia.org/license/patent.
+
+#![allow(non_snake_case)]
```

Comment:
Why,
can we have in SnakeCase itself?
It would be better to have consistent styling

Reply:
The reason for snake casing is, that it is the standard for JavaScript and wasm-bindgen just takes the names over.